### ISSUE TITLE
Interactions: adding a comment line results in an error

### ISSUE THREAD

**Issue by TitouanReal** [gaphor/gaphor#2452]
### Describe the bug
Trying to add a comment line can fail on specific conditions.

### To Reproduce
1. Create a Generic empty model.
2. Add an interaction.
3. Add a lifeline to the interaction.
4. Add an execution specification to the lifeline.
5. Add a comment.
6. Try to add a comment line.
7. Trying to add the comment line immediately fails on the first click of the drag.

I can't add the Gaphor error report in this issue due to the character limit. Please let me know how I can provide this information if necessary.

### Expected behavior
I expect the comment line to draw.

### OS
- [x] Linux (Fedora 38)
- [ ] macOS
- [ ] Windows

### Version
Version of Gaphor: 2.19.2

**Comment by amolenaar** [comment]
I can trigger an issue. I hope it's the same that you see in the Error Reporter:


```
Time since application startup: 0:00:48
|Traceback (most recent call last):
|  File "/home/arjan/Development/gaphor/.venv/lib/python3.11/site-packages/gaphas/view/gtkview.py", line 506, in do_snapshot
|    self.painter.paint(list(items), cr)
|    │                       │       └ <cairo.Context object at 0x7f8de995a590>
|    │                       └ <generator object Diagram.sort.<locals>.<genexpr> at 0x7f8de9976f80>
|    └ <gtkview.GtkView object at 0x7f8dc1bedf40 (GaphasView at 0x55837025b7d0)>
|  File "/home/arjan/Development/gaphor/.venv/lib/python3.11/site-packages/gaphas/painter/chain.py", line 30, in paint
|    painter.paint(items, cairo)
|    │             │      └ <cairo.Context object at 0x7f8de995a590>
|    │             └ [<gaphor.UML.actions.activity.ActivityItem element 8c3b4530-1ff9-11ee-a057-0456e5e540ed>, <gaphor.UML.interactions.interaction.I...
|    └ <gaphor.diagram.painter.ItemPainter object at 0x7f8dc1bc3a50>
|  File "/home/arjan/Development/gaphor/gaphor/diagram/painter.py", line 53, in paint
|    self.paint_item(item, cr)
|    │               │     └ <cairo.Context object at 0x7f8de995a590>
|    │               └ <gaphor.UML.interactions.executionspecification.ExecutionSpecificationItem element 969c15ae-1ff9-11ee-a057-0456e5e540ed>
|    └ <gaphor.diagram.painter.ItemPainter object at 0x7f8dc1bc3a50>
|  File "/home/arjan/Development/gaphor/gaphor/diagram/painter.py", line 28, in paint_item
|    style = diagram.style(StyledItem(item, selection, self.dark_mode))
|            │             │          │     │          └ <gaphor.diagram.painter.ItemPainter object at 0x7f8dc1bc3a50>
|            │             │          │     └ <gaphor.diagram.selection.Selection object at 0x7f8dc1beee90>
|            │             │          └ <gaphor.UML.interactions.executionspecification.ExecutionSpecificationItem element 969c15ae-1ff9-11ee-a057-0456e5e540ed>
|            │             └ <class 'gaphor.core.modeling.diagram.StyledItem'>
|            └ <gaphor.core.modeling.diagram.Diagram element 58d6c536-66f8-11ec-b4c8-0456e5e540ed>
|  File "/home/arjan/Development/gaphor/gaphor/core/modeling/diagram.py", line 281, in style
|    return style_sheet.match(node) if style_sheet else FALLBACK_STYLE
|           │                 │        │                └ {'color': (0, 0, 0, 1), 'font-family': 'sans', 'font-size': 14, 'line-width': 2, 'padding': (0, 0, 0, 0)}
|           │                 │        └ <gaphor.core.modeling.stylesheet.StyleSheet element 58d6989a-66f8-11ec-b4c8-0456e5e540ed>
|           │                 └ <gaphor.core.modeling.diagram.StyledItem object at 0x7f8de9950ad0>
|           └ <gaphor.core.modeling.stylesheet.StyleSheet element 58d6989a-66f8-11ec-b4c8-0456e5e540ed>
|  File "/home/arjan/Development/gaphor/gaphor/core/modeling/stylesheet.py", line 106, in match
|    return self._compiled_style_sheet.match(node)
|           │                                └ <gaphor.core.modeling.diagram.StyledItem object at 0x7f8de9950ad0>
|           └ <gaphor.core.modeling.stylesheet.StyleSheet element 58d6989a-66f8-11ec-b4c8-0456e5e540ed>
|  File "/home/arjan/Development/gaphor/gaphor/core/styling/__init__.py", line 137, in match
|    return merge_styles(*(decl for _, _, decl in results))  # type: ignore[arg-type]
|           │                                     └ [((0, 0, 0), 0, {'--opaque-background-color': 'white', 'background-color': (0.0, 0.0, 0.0, 0.0), 'color': (0.0, 0.0, 0.0, 1.0), ...
|           └ <function merge_styles at 0x7f8e3aae0b80>
|  File "/home/arjan/Development/gaphor/gaphor/core/styling/__init__.py", line 88, in merge_styles
|    if color and color[3] > 0.0:
```

This is a regression introduced in 2.19.2, as a result of #2439. ... I think ...

**Comment by TitouanReal** [comment]
It looks like it's the same error that I got.
I saw you introduced a change to fix it. Thanks!

**Comment by amolenaar** [comment]
Fixed by #2460.
